modules/library/referencing/src/main/java/org/geotools/measure/SexagesimalConverter.java

```diff
@@ -37,7 +35,7 @@
  */
 class SexagesimalConverter extends AbstractConverter {
     /** Serial number for compatibility with different versions. */
-    private static final long serialVersionUID = 3873494343412121773L;
+    private static final long serialVersionUID = -2663951106460584999L;
```

Comment:
Shy is the `serialVersionUID` changing?

Reply:
I tried to change as little as possible but the parent this class extends is different.  It appears they move the AbstractConverter from one of their Unit System libraries to their reference implementation.

Wouldn't the class changing its hierarchy mean the serialVersionUID should change?  Your judgement is most likely better then mine but just want to point out the parent class has changed.

Reply:
Understood, thanks for the clarification.